kinematic_properties.py

```diff
@@ -171,10 +171,64 @@ def get_projected_inertia_tensor(mass, position, axis):
         np.array((Itensor[0, 0], Itensor[1, 1], Itensor[0, 1]))
         * position.units
         * position.units
-        * mass.units
     )
     return Itensor
 
+def get_reduced_inertia_tensor(mass, position):
```

Comment:
There is quite a lot of duplicated code shared with get_inertia_tensor. Could you combine these functions by passing a keyword argument, e.g. `get_inertia_tensor(mass, position, reduce=False)`

Reply:
Yes, I will make the changes